BUILD.md

```diff
@@ -1,5 +1,10 @@
 # Build instructions
 
+## Table of Contents
+* [Windows](#windows)
```

Comment:
Headers and lists should have one blank line separating them from the other content, see [MD022](https://github.com/DavidAnson/markdownlint/blob/main/doc/md022.md) and [MD032](https://github.com/DavidAnson/markdownlint/blob/main/doc/md032.md)